pptk/viewer/viewer.py

```diff
@@ -398,6 +400,9 @@ def record(self, folder, poses, ts=[], tlim=[-numpy.inf, numpy.inf],
                 struct.pack('2f', t, t) + \
                 struct.pack('?', False)
             self.__send(msg)
+            # give viewer time to finish render
+            if shutter_speed != numpy.inf:
```

Comment:
Can it be changed to    "shutter_speed : Optional[numpy.int]=None" and check if shutter_speed is not None:

Reply:
I've changed it. Let hope that's ok now.